tests/UI/campaigns/functional/BO/13_shopParameters/02_orderSettings/02_statuses/returnStatuses/02_filterSortAndPagination.js

```diff
@@ -0,0 +1,302 @@
+require('module-alias/register');
+
+// Helpers to open and close browser
+const helper = require('@utils/helpers');
+
+// Common tests login BO
+const loginCommon = require('@commonTests/loginBO');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const orderSettingsPage = require('@pages/BO/shopParameters/orderSettings');
+const statusesPage = require('@pages/BO/shopParameters/orderSettings/statuses');
+const addOrderReturnStatusPage = require('@pages/BO/shopParameters/orderSettings/statuses/returnStatus/add');
+
+// Import data
+const {ReturnStatuses} = require('@data/demo/orderReturnStatuses');
+const OrderReturnStatusFaker = require('@data/faker/orderReturnStatus');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_shopParameters_orderSettings_statuses_'
+  + 'returnStatuses_filterSortAndPaginationOrderStatus';
```

Comment:
I need to check the database to see if I allowed enough characters... :upside_down_face: